.github/release-please-config.json

```diff
@@ -1,15 +1,15 @@
 {
   "bootstrap-sha": "24485f3dd799664850813a7dfcc31d6043d8bf47",
   "packages": {
-    "./packages/RdfService/": {
+    "packages/RdfService/": {
```

Comment:
Trailing / probably not required, `./`... was deemed invalid.
https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md